packages/iov-core/src/signingservercore.spec.ts

```diff
@@ -356,7 +349,7 @@ describe("SigningServerCore", () => {
         await sendTokensFromFaucet(connection, identity, minimalFee);
       }
 
-      async function throwingCallback(_1: string, _2: UnsignedTransaction): Promise<boolean> {
+      async function throwingCallback(): Promise<boolean> {
```

Comment:
To be honest, this those missing arguments, I find it much harder to see why the callbacks actually work for the interface. Also see the other option below